src/components/Navigation/ChromeNavItem.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { useEffect } from 'react';
-import { NavItem } from '@patternfly/react-core';
+import { Icon, NavItem } from '@patternfly/react-core';
 import ExternalLinkAltIcon from '@patternfly/react-icons/dist/js/icons/external-link-alt-icon';
+import FlaskIcon from '@patternfly/react-icons/dist/js/icons/flask-icon';
 import BellIcon from '@patternfly/react-icons/dist/js/icons/bell-icon';
 import { titleCase } from 'title-case';
 import classNames from 'classnames';
 import get from 'lodash/get';
 
 import { isBeta } from '../../utils/common';
-import { betaBadge } from '../Header/Tools';
```

Comment:
Looks like the `betaBadge` is no longer used anywhere else in the code. Can you remove the function source as well?

Reply:
@Hyperkid123 Isn't it still in the masthead?